action.yml

```diff
@@ -54,11 +58,12 @@ runs:
   using: 'composite'
   steps:
     - run: |
+        # Override GITHUB_SHA if we have inputs.sha
```

Comment:
We would likely need to rename this instead of modifying the default.

Suggestion:

```yaml
      env:
        GITHUB_BASE_REF: ${{ github.base_ref }}
        GITHUB_REPOSITORY: ${{ github.repository }}
        # INPUT_<VARIABLE_NAME> is not available in Composite run steps
        # https://github.community/t/input-variable-name-is-not-available-in-composite-run-steps/127611
        INPUT_SHA: ${{ inputs.sha }}
        INPUT_TOKEN: ${{ inputs.token }}
        INPUT_FILES: ${{ inputs.files }}
        INPUT_SEPARATOR: ${{ inputs.separator }}
```

This would require updating the `entrypoint.sh` renaming all usage of `GITHUB_SHA` -> `INPUT_SHA`

Reply:
This approach is better :+1:  I like it :) . Done!

Reply:
Thanks